tox.ini

```diff
@@ -1,5 +1,5 @@
 [tox]
-envlist = pep8,mypy-py{36,37,38,39,310},py{36,37,38,39,310},coverage,docs
+envlist = pep8,mypy-py{36,37,38,39,310},py{36,37,38,39,310},coverage,sphinx-docs,py-docs,check-py-docs
```

Comment:
It doesn't make much sense to include both `py-docs` and `check-py-docs` in `envlist` (i.e. the stuff that's executed on simple `tox` invocation with no `-e`)... I'd leave just `py-docs`

Reply:
@PawelLipski Hmm, i think it has to stay since modified Ci for docs executes both `check-py-docs` and `sphinx-docs`

Reply:
But does an env need to be included in `envlist` to be executable? Doesn't `envlist` just control the behavior of no-argument `tox` (which is never called this way in CI, IIRC)?